ES6/index.js

```diff
@@ -0,0 +1,74 @@
+import {
+  navBar, listLink, addLink, contactLink,
+} from '../modules/navBar.js';
+// import { Book } from "../modules/addRemoveBook.js";
```

Comment:
- [ ] Kindly remove all commented out and unused/ unwanted pieces of code from your code base. It makes it look cluttered and messy.